event/consumer_test.go

```diff
@@ -21,11 +24,13 @@ func TestConsume_UnmarshallError(t *testing.T) {
 
 		messages <- kafkatest.NewMessage([]byte("invalid schema"))
 		messages <- kafkatest.NewMessage(Marshal(*expectedEvent))
-		close(messages)
 
 		Convey("When consume messages is called", func() {
 
-			event.Consume(messageConsumer, handler)
+			consumer := event.NewConsumer()
+			go consumer.Consume(messageConsumer, handler)
```

Comment:
The Consume method wraps everything in a go routine, so probably don't need `go` before this function call (throughout this file)

Reply:
Good spot - this was a left over from refactoring.